tests/libtest.sh

```diff
@@ -73,7 +73,7 @@ cleanup() {
     outfile=$4
   fi
 
-
+  rm -rf /var/lib/container-storage-setup
```

Comment:
I don't like this, we need a way to override this directory, so that the tests are not descructive.

At lease only remove

/var/lib/container-storage-setup/container